prow/config/labels.yaml

```diff
@@ -151,6 +151,12 @@ default:
       target: prs
       prowPlugin: lgtm
       addedBy: reviewers or members
+    - color: 800000
+      description: Tests a PR with Metal3 CI
+      name: CI
+      target: prs
+      prowPlugin: label
+      addedBy: reviewers or members
```

Comment:
I don’t think we want this on all repos. Just the ironic image ones, and they already have the label.

Reply:
I was hoping `addedBy` restricts who can add a certain label, so not anyone can trigger CI

Reply:
AFAICT there's no access control around "/label", so maybe we shouldn't enable this at all

https://github.com/kubernetes/test-infra/blob/master/prow/plugins/label/label.go